### PR TITLE
Website 2.0

### PR DESCRIPTION
**Pull request by darrylmasson**
Extensive changes to every file, focused mainly on unifying style, removing scaffolding, and some work towards #8.

### CODE REVIEW

File: public/javascripts/status_script.js [hunk 1]
```diff
@@ -1,431 +1,386 @@
...
     } else {
-      $("#dispatcher_status").html("offline");
-      $("#dispatcher_status").css("color", "red");
+      $("#dispatcher_status").html("offline").css("color", "red");
     }
   });
 }
 
+function UnpackEBStatus(doc) {
+  if (doc.state === 'busy') return ['green', `processing ${doc.runid} on ${doc.max_cores} cores`];
```
Comment (jorana) [inline public/javascripts/status_script.js:148]: @darrylmasson could we re-add the targets here or is the tuple of targets confusing?

**Review by jorana — COMMENTED** [review]
I like the new website :tada:!

File: public/javascripts/status_script.js (hunk 1 above)
Comment (jorana) [reply public/javascripts/status_script.js:148]: The reason I ask is that e.g. it shows quite nicely which runs are nv/mv etc.

Comment (darrylmasson) [reply public/javascripts/status_script.js:148]: This shouldn't be difficult

**Pull request merged**